projects/demo/src/app/video/video.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { YouTubePlayerModule } from '@angular/youtube-player';
+import { VideoComponent } from './video.component';
+
+describe('VideoComponent', () => {
+  let component: VideoComponent;
+  let fixture: ComponentFixture<VideoComponent>;
+
+  beforeEach(async () => {
```

Comment:
issue: If we're actually writing tests as part of the content, leave them in but migrate `async` to `waitForAsync`. If we're not writing tests as part of the content, delete them.